core/domain/feedback_services.py

```diff
@@ -139,81 +139,142 @@ def create_message(
         received_via_email: bool. Whether new message is received via email or
             web.
     """
+    create_messages(
+        [thread_id], author_id, updated_status, updated_subject, text,
+        received_via_email=received_via_email)
 
+
+def create_messages(
+        thread_ids, author_id, updated_status, updated_subject, text,
+        received_via_email=False):
+    """Creates a new message for each of the threads in thread_ids and
+    for each message, subscribes the author to the thread.
+
+    Args:
+        thread_ids: list(str). The thread ids the messages belong to.
+        author_id: str. The author id who creates the messages.
+        updated_status: str. One of STATUS_CHOICES. New thread status.
+            Must be supplied if this is the first message of the threads.
```

Comment:
This is a bit confusing. Is the same updated status to be applied to all threads? If so, this should be stated more clearly.

Ditto for updated_subject and text. It should be clear that these properties will apply to all messages.